dango/plugins/fun.py

```diff
@@ -209,7 +209,7 @@ def _gifmap(avys):
 
         start = time.time()
         async with ctx.typing():
-            avys = await asyncio.gather([fetch_image(t.avatar_url_as(format="png")) for t in [source + list(dest)]])
+            avys = await asyncio.gather(*[fetch_image(t.avatar_url_as(format="png")) for t in [source] + list(dest)])
```

Comment:
pls squash this onto whatever introduced the typo

Reply:
Squashed it